client/db_test.go

```diff
@@ -245,3 +245,29 @@ func ExampleDB_Insecure() {
 	// Output:
 	// aa=1
 }
+
+func ExampleDB_Open() {
```

Comment:
> (do examples that don't fit the pattern even get run?)

They do, I've double-checked mine.

> It's fine to just do a simple test here to make sure that Open doesn't return an error or panic.

What about something [like this](https://gist.github.com/kharin/24224ca23a662e39a82c)?